src/screens/regimen/regimen_routine/styles.ts

```diff
@@ -0,0 +1,118 @@
+import styled from 'styled-components/native';
+import Constants from 'expo-constants';
+
+import applyScale from '../../../utils/applyScale';
+
+export const SplashTopStatusBar = styled.View`
+  width: 100%;
+  height: ${Constants.statusBarHeight}px;
+  position: absolute;
```

Comment:
Always try using flex to arrange component in other  take full advantage of responsiveness across all screen sizes.